test/DistributionCreator.t.sol

```diff
@@ -38,115 +48,1627 @@ contract DistributionCreatorTest is Fixture {
+                rewardToken: address(rewardToken),
+                amount: amount,
+                campaignType: 1,
+                startTimestamp: startTimestamp,
+                duration: 3600 * 24,
+                campaignData: abi.encode(
+                    0xbEEfa1aBfEbE621DF50ceaEF9f54FdB73648c92C,
+                    new address[](0),
+                    new address[](0),
+                    "",
+                    new bytes[](0),
+                    new bytes[](0),
+                    hex""
+                )
+            })
+        );
+
+        vm.prank(governor);
+        // Create false tree
+        distributor.updateTree(
+            MerkleTree({
+                merkleRoot: bytes32(0x0b70a97c062cb747158b89e27df5bbda859ba072232efcbe92e383e9d74b8555),
+                ipfsHash: keccak256("IPFS_HASH")
+            })
+        );
+
+        angle.mint(address(distributor), 1e18);
+        agEUR.mint(address(distributor), 5e17);
+
+        vm.warp(distributor.endOfDisputePeriod() + 1);
+        {
+            bytes32[][] memory proofs = new bytes32[][](1);
+            address[] memory users = new address[](1);
+            address[] memory tokens = new address[](1);
+            uint256[] memory amounts = new uint256[](1);
+            // proofs[0] = new bytes32[](1);
+            // proofs[0][0] = bytes32(0x6f46ee2909b99367a0d9932a11f1bdb85c9354480c9de277d21086f9a8925c0a);
+            // users[0] = alice;
+            // tokens[0] = address(angle);
+            // amounts[0] = 1e18;
+            proofs[0] = new bytes32[](1);
+            proofs[0][0] = bytes32(0x3a64e591d79db8530701e6f3dbdd95dc74681291b327d0ce4acc97024a61430c);
+            users[0] = bob;
+            tokens[0] = address(agEUR);
+            amounts[0] = 5e17;
+
+            uint256 aliceBalance = angle.balanceOf(address(alice));
+            uint256 bobBalance = agEUR.balanceOf(address(bob));
+
+            vm.prank(bob);
+            distributor.claim(users, tokens, amounts, proofs);
+        }
+
+        {
+            address[] memory users = new address[](1);
+            users[0] = alice;
+
+            vm.prank(bob);
+            vm.expectRevert(Errors.InvalidOverride.selector);
+            creator.reallocateCampaignRewards(campaignId, users, address(governor));
+
+            assertEq(creator.campaignReallocation(campaignId, alice), address(0));
+        }
+
+        {
+            address[] memory users = new address[](1);
+            users[0] = alice;
+
+            vm.prank(alice);
+            creator.reallocateCampaignRewards(campaignId, users, address(governor));
+
+            assertEq(creator.campaignReallocation(campaignId, alice), address(governor));
+
+            address[] memory listReallocation = creator.getCampaignListReallocation(campaignId);
+            assertEq(listReallocation.length, 1);
+            assertEq(listReallocation[0], alice);
+        }
+    }
+}
+
+// contract DistributionCreatorOverrideTest is Fixture {
```

Comment:
why is it commented?

Reply:
Uncommented and stored variables in storage rather than in stack, making the test pass even without `--via-ir`.